### PR TITLE
[1.2.1]: Fix deploy script and allow to clear data

### PR DESCRIPTION
**Pull request by meatball133**

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -1,12 +1,8 @@
 name: Docs
+on: [push]
```
Comment (ErikSchierboom) [inline .github/workflows/docs.yml:2]: But wouldn't this overwrite the  docs whenever a PR is opened? That would mean that the docs are potentially temporarily incorrect

Comment (meatball133) [reply .github/workflows/docs.yml:2]: Yeah I will fix so it only generates new docs on push to the main branch.

Comment (ErikSchierboom) [reply .github/workflows/docs.yml:2]: 👍

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -1,5 +1,11 @@
 name: Docs
-on: [push]
+on:
+  pull_request:
```
Comment (ErikSchierboom) [inline .github/workflows/docs.yml:3]: I don't think you want the `pull_request` bit. What it says (IIRC) is to run when a pull request is opened that targets the `main` branch. I think you just want the `push` part.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**